content/en/docs/concepts/traffic-management/index.md

```diff
@@ -309,8 +309,8 @@ spec:
 For some match conditions, you can also choose to select them using the exact
 value, a prefix, or a regex.
 
-You can add multiple match conditions to the same `match` block to AND your
-conditions, or add multiple match blocks to the same rule to OR your conditions.
+You can add multiple match conditions to the same `match` block to OR your
+conditions, or add multiple match blocks to the same rule to AND your conditions.
```

Comment:
This change is incorrect. The documentation is correct as written.

AND example (1 match block, 2 conditions):
```
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  name: ratings
spec:
  hosts:
  - ratings
  http:
  - match:
    - sourceLabels:
        app: reviews
        version: v2
      headers:
        end-user:
          exact: jason
```

OR example (2 match blocks):
```
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  name: ratings
spec:
  hosts:
  - ratings
  http:
  - match:
    - sourceLabels:
        app: reviews
        version: v2
    - headers:
        end-user:
          exact: jason
```